### ISSUE TITLE
Build issue with gcc 6

### ISSUE THREAD

**Issue by tillea** [bcgsc/abyss#131]
Hi,

when building the abyss Debian package with gcc version 6.1 I get the following error:

```
g++ -Wall -Wextra -pthread -Wno-error -fopenmp -g -O2 -fdebug-prefix-map=/build/abyss-2.0.1=. -fstack-protector-strong -Wformat -Werror=format-security  -L/usr/lib/openmpi/lib   -Wl,-z,    relro -o Konnector_konnector Konnector/Konnector_konnector-konnectorTest.o ../Align/libalign.a ../Common/libcommon.a ../lib/gtest-1.7.0/libgtest_main.a -ldl -lm
g++ -DHAVE_CONFIG_H -I. -I..  -I.. -I../lib/gtest-1.7.0/include -I../DataLayer -I../Common -I/build/abyss-2.0.1  -isystem/usr/lib/openmpi/include   -Wdate-time -D_FORTIFY_SOURCE=2 -isystem/build/abyss-2.0.1/boost_1_56_0 -Wall -Wextra -pthread -Wno-error -fopenmp -g -O2 -fdebug-prefix-map=/build/abyss-2.0.1=. -fstack-protector-strong -Wformat -Werror=format-security -c -o DBG/DBG_LoadAlgorithm-LoadAlgorithmTest.o `test -f 'DBG/LoadAlgorithmTest.cpp' || echo './'`DBG/LoadAlgorithmTest.cpp
In file included from ../lib/gtest-1.7.0/include/gtest/internal/gtest-internal.h:40:0,
                 from ../lib/gtest-1.7.0/include/gtest/gtest.h:58,
                 from DBG/LoadAlgorithmTest.cpp:7:
../lib/gtest-1.7.0/include/gtest/internal/gtest-port.h:557:14: error: 'tuple' is already declared in this scope
 using ::std::tuple;
              ^~~~~
../lib/gtest-1.7.0/include/gtest/internal/gtest-port.h:558:14: error: 'tuple_element' is already declared in this scope
 using ::std::tuple_element;
              ^~~~~~~~~~~~~
../lib/gtest-1.7.0/include/gtest/internal/gtest-port.h:559:14: error: 'tuple_size' is already declared in this scope
 using ::std::tuple_size;
              ^~~~~~~~~~
Makefile:1272: recipe for target 'DBG/DBG_LoadAlgorithm-LoadAlgorithmTest.o' failed
make[3]: *** [DBG/DBG_LoadAlgorithm-LoadAlgorithmTest.o] Error 1
make[3]: Leaving directory '/build/abyss-2.0.1/Unittest'
```

Kind regards  Andreas.

**Comment by sjackman** [comment]
Thanks for the bug report, Andreas. We'll look into it and hopefully have a fix for 2.0.2.

**Comment by sjackman** [comment]
Hi, Ben. This issue came up with one of the participants at Hackseq this  weekend.

**Comment by benvvalk** [comment]
fixed by 184b2918 and 0cf8ed729
